docs/install/index.md

```diff
@@ -176,27 +176,112 @@ Download the [onnxruntime-android](https://mvnrepository.com/artifact/com.micros
 
 Refer to the instructions for creating a [custom Android package](../build/custom.md#android).
 
-## ORT Training package
+## Install for On-Device Training
 
+Unless stated otherwise, the installation instructions in this section refer to pre-built packages designed to perform on-device training.
+
+If the pre-built training package supports your model but is too large, you can create a [custom training build](../build/custom.md).
+
+### Offline Phase - Prepare for Training
+
+```bash
+pip install onnxruntime-training
 ```
+
+### Training Phase - On-Device Training
+
+<table>
+  <tr>
+    <th>Device</th>
+    <th>Language</th>
+    <th>PackageName</th>
+    <th>Installation Instructions</th>
+  </tr>
+  <tr>
+    <td>Windows</td>
+    <td>C, C++, C#</td>
+    <!-- TODO (baijumeswani) - Update to Training link once published -->
+    <td><a href="https://www.nuget.org/packages/Microsoft.ML.OnnxRuntime">Microsoft.ML.OnnxRuntime.Training</a></td>
+    <td>
+      <pre lang="bash">dotnet add package Microsoft.ML.OnnxRuntime.Training</pre>
+    </td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>Python</td>
+    <td><a href="https://pypi.org/project/onnxruntime-training/">onnxruntime-training</a></td>
```

Comment:
we dont have a python wheel in pypi for windows

Reply:
We don't have a stable windows wheel for training anywhere at the moment. We only have one for cuda nightly.

I was hoping that for this release we could do a manual windows cpu whl upload to pypi.